test/unit/widgets/buttons.test.ts

```diff
@@ -105,7 +111,14 @@ describe('button', () => {
     const text = 'txt'
     const handler = () => {
     }
-    expect(button(dom, iconURI, text, handler)).toBeTruthy()
+    // FIXME: Not sure why this is needed, but
+    // JSS tries to do .insertBefore on a StyleSheet element,
+    // and that seems to fail in JSDOM (it works fine in the browser)
+    const domCopy = {
```

Comment:
Probably want a helper for this quickly so that we can easily get a DOM-object with head for other tests as well.